requirements/docker.txt

```diff
@@ -12,17 +12,17 @@ django-redis-cache==2.1.3  # pyup: ignore
 pillow==8.1.0
 
 # local debugging tools
-watchdog==2.0.0
+watchdog==2.0.1
 datadiff==2.0.0
 ipdb==0.13.4
 pdbpp==0.10.2
 
 # jedi 0.18 is incompatible with ipython
 # https://github.com/ipython/ipython/issues/12740
-jedi>0.17,<0.18
+jedi==0.18.0
```

Comment:
Hrm... seems we didn't ignore this. @stsewd did you test if the problem with ipython disappeared? We may need to pin it back, otherwise.

Reply:
opps, thought this was <=18.0, I don't use ipython, so not sure about this problem, but also I think shell_plus depends on ipython.

Reply:
Yeah, shell_plus uses it. We need to revert this change for now and add the `# pyup: ignore` until it gets fixed upstream.